source/manuals/accessibility.html.md.erb

```diff
@@ -0,0 +1,115 @@
+---
+title: Building accessible services
+last_reviewed_on: 2019-06-11
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Government services should be accessible to everyone. This includes anyone with a visual, motor or learning impairment but also includes anyone with a temporary or situational disability, such as a broken arm or working in a loud environment.
+
+Building a service with accessibility in mind not only allows those with access needs to use your service, it also improves the service for everyone else. An accessibility problem with a website can be something that affects everyone, not just people who can only access the web with a keyboard or screen reader.
+
+Government services are legally required to be accessible. This means services must comply with the international WCAG 2.1 AA accessibility standard. New regulation means that existing sites and apps must be compliant by 2020.
+
+Further reading:
+
+- [legal accessibility requirements for government services](https://gds.blog.gov.uk/2018/09/24/how-were-helping-public-sector-websites-meet-accessibility-requirements/)
+- [the equality act](https://www.gov.uk/guidance/equality-act-2010-guidance)
+- [understanding WCAG 2.1](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
+
+## How to make your service accessible
+
+### Consider accessibility from the start
+
+Accessibility cannot be achieved by adding some final touches; it needs to be considered at all stages of a project. Designs should be reviewed for possible issues, tests should be written and run throughout development, services should be tested with accessibility in mind.
+
+### Understand that not everyone reads content the same way
+
+A sighted user might navigate a page from top to bottom, perhaps skim reading through headings and paragraphs to find the content they want.
+
+Non sighted users can also skim read a page. Screen readers can announce content by element type, such as headings, paragraphs or links. For example, if a screen reader user expects a page to contain data in a table element, such as a train timetable, they might start by reading through all the tables on a page.
+
+This is why semantic markup and good heading structure are important when building accessible services.
+
+### Automated testing
+
+Accessibility testing tools such as [aXe](https://www.deque.com/axe/) and [Lighthouse](https://developers.google.com/web/tools/lighthouse/) are useful for finding issues, but [automated testing can only find around 30% of likely accessibility problems](https://alphagov.github.io/accessibility-tool-audit/). Automated testing shouldn’t be the only part of accessibility testing a service.
```

Comment:
This sentence feels incomplete: "Automated testing shouldn’t be the only part of accessibility testing a service"

Maybe?:
"Automated testing shouldn't be the only accessibility testing a service does. You should also test with real users."

Reply:
I'll change it, thanks.